api/proto/config/devmodel.proto

```diff
@@ -34,16 +57,23 @@ message sWAdapterParams {
 	repeated string bondgroup = 10;
 }
 
+// systemAdapters, are the higher l2 concept built on physicalIOs.
+// systemAdapters, gives all the required bits to turn the physical IOs
+// into useful IP endpoints
+// These endpoints can be further used to connect to controller or 
+// can be shared between workload/services running on the node.
 message SystemAdapter {
 	// name of the adapter; hardware-specific e.g., eth0
 	string name = 1;
 
 	sWAdapterParams allocDetails = 20;
 
 	// this is part of the freelink group
+	// deprecate: look at PhysicalIO->Usage
```

Comment:
Note that we don't have a free thing in usage. I think we should add a PhysicalIO->Policy or a SystemAdapter->Policy which is where we can start with the "free" bool and potentially add others down the road.

Reply:
I am saying that we have that policy already in Usage field. We don't need anything else. Policy can be better name, if you like I can rename to to it.